velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -60,20 +61,132 @@ void WindowFuzzer::addWindowFunctionSignatures(
   }
 }
 
-std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
-  auto frameType = [](int value) -> const std::string {
-    switch (value) {
-      case 0:
-        return "RANGE";
-      case 1:
-        return "ROWS";
-      default:
-        VELOX_UNREACHABLE("Unknown value for frame type generation");
-    }
+std::tuple<
+    core::WindowNode::WindowType,
+    core::WindowNode::BoundType,
+    core::WindowNode::BoundType>
+WindowFuzzer::frameWindowTypeAndBoundType() {
```

Comment:
generateFrameClause was a better name for this method. Is it possible to retain it ?